src/tdastro/astro_utils/passbands.py

```diff
@@ -0,0 +1,220 @@
+import os
+import time
+import urllib.request
+from pathlib import Path
+from urllib.error import HTTPError, URLError
+
+import numpy as np
+import scipy.integrate
+
+from tdastro.sources.physical_model import PhysicalModel
+
+
+class Passbands:
+    """Class to handle the transmission tables and normalization for different passbands."""
+
+    def __init__(self, bands=None):
+        if bands is None:
+            self.bands = ["u", "g", "r", "i", "z", "y"]
+        else:
+            self.bands = bands
+        self.data_path = None
+        self.transmission_tables = {}
+        self.normalized_system_response_tables = {}
```

Comment:
It would be nice if you can add these attributes to the class docstring to document units.